### PR TITLE
feat(inputs.disk): add label as tag

### PR DESCRIPTION
**Pull request by powersj**
fixes: #12594

### CODE REVIEW

**Review by Hipska — CHANGES_REQUESTED** [review]
Please also update the README sections Metrics and Example Output.

**Review by Hipska — APPROVED** [review]

File: plugins/inputs/disk/disk.go [hunk 1]
```diff
@@ -53,13 +54,24 @@ func (ds *DiskStats) Gather(acc telegraf.Accumulator) error {
...
 			"mode":   mountOpts.Mode(),
 		}
+
+		m := strings.Count(device, "/dev/")
+		if m > 0 {
+			label, _ := disk.Label(strings.Replace(device, "/dev/", "", 1))
+			if label != "" {
+				tags["label"] = label
+			}
+		}
```
Comment (srebhan) [inline plugins/inputs/disk/disk.go:73]: Wouldn't this be more clear
```suggestion
		label, err := disk.Label(strings.TrimPrefix(device, "/dev/"))
		if err == nil && label != "" {
			tags["label"] = label
		}
```

**Review by srebhan — COMMENTED** [review]
@powersj one small suggestion... Can you please take a look!?

File: plugins/inputs/disk/disk.go (hunk 1 above)
Comment (powersj) [reply plugins/inputs/disk/disk.go]: agreed, committed

**Review by Hipska — APPROVED** [review]

**Pull request merged**